<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMenuEditUI</class>
 <widget class="QDialog" name="DlgMenuEditUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>391</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="menuGbx">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>201</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="menuLed"/>
    </item>
    <item>
     <widget class="QLabel" name="menuItemsLbl">
      <property name="text">
       <string>Menu Items:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="menuEditLed"/>
    </item>
    <item>
     <widget class="QListWidget" name="menuEditLstWdt"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="commandGbx">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>202</width>
     <height>254</height>
    </rect>
   </property>
   <property name="title">
    <string>Command</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="fileLbl">
      <property name="text">
       <string>File Program:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="fileLed"/>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="fileBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="paramMenuLbl">
      <property name="text">
       <string>Parameters:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="paramMenuLed"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="shortcut">
      <property name="text">
       <string>Shortcut:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="shortcutCbx">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxVisibleItems">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="saveModelBefore">
      <property name="text">
       <string>Save Model Before</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QCheckBox" name="needsOpenModel">
      <property name="text">
       <string>Needs an Open Model</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="toolbarGbx">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>201</width>
     <height>63</height>
    </rect>
   </property>
   <property name="title">
    <string>Toolbar Icon</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="selectBtn">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeBtn">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="iconLbl">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="addMenuGbx">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>440</y>
     <width>202</width>
     <height>63</height>
    </rect>
   </property>
   <property name="title">
    <string>Menu Item</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="addBtn">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="removeItemBtn">
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgMenuEditUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgMenuEditUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
